docs: sd-step-group docs [skip chromatic]

smfonseca opened the pull request:
<!-- ## Title: Please consider adding the [skip chromatic] flag to the PR title in case you dont need chromatic testing your changes. -->
## Description:
<!-- *PR notes: Please describe the changes in this PR.* -->

## Definition of Reviewable:
<!-- *PR notes: Irrelevant elements should be removed.* -->
- [ ] Documentation is created/updated
- [ ] Migration Guide is created/updated
- [ ] E2E tests (features, a11y, bug fixes) are created/updated
- [ ] Stories (features, a11y) are created/updated
- [ ] relevant tickets are linked

mariohamann commented:
Could we make it more "speaking" in the examples?

This would be one for `non-interactive` (including icons):

```
1. Termin vereinbaren
Lassen Sie sich von den Experten in unseren Partnerbanken beraten und finden Sie den richtigen Fondssparplan.
2. Fonds auswählen & Fondssparplan abschließen
Wählen Sie aus einer Vielzahl von Fonds den passenden für Ihren Fondssparplan aus.
3. Depot eröffnen
Eröffnen Sie gemeinsam mit Ihrem Bankberater Ihr eigenes Wertpapierdepot.
```

![CleanShot 2024-08-09 at 11 06 22](https://github.com/user-attachments/assets/6a5c58eb-96bf-4e52-bb40-a4e54f5a3c1e)

And maybe for non-interactive we only use the "headlines":
```
1. Termin vereinbaren
2. Fonds auswählen & Fondssparplan abschließen
3. Depot eröffnen
```

Ideally in English – but I have no idea regarding fitting translations here. 😆

MartaPintoTeixeira commented on packages/components/src/components/step-group/step-group.stories.ts:
```diff
@@ -1,15 +1,24 @@
 const { generateTemplate } = storybookTemplate('sd-step-group');
 
+/**
+ * Used as navigation bar that guides users through the steps of a process or task
```
test

Vahid1919 commented:
@smfonseca I did not 100% understand the feedback you received here. If you are more aligned can you take over this PR?

coraliefeil commented on packages/components/src/components/step-group/step-group.stories.ts:
```diff
@@ -1,15 +1,24 @@
+ * Used as navigation bar that guides users through the steps of a process or task
+ *
+ * **Related templates:**
+ * - [sd-step-group](?path=/docs/templates-step-group--docs)
```
Step Group

smfonseca replied:
Updated

coraliefeil commented on packages/components/src/components/step-group/step-group.stories.ts:
```diff
@@ -1,15 +1,24 @@
+  tags: ['!dev'],
   component: 'sd-step-group',
   args: overrideArgs([
     {
```
Show:

![image](https://github.com/user-attachments/assets/24e69fbc-4046-464e-b2cb-8128adb05938)

smfonseca replied:
Updated

coraliefeil commented on packages/components/src/components/step-group/step-group.stories.ts:
```diff
@@ -44,240 +53,184 @@ export default {
-  ]
-};
+  name: 'Orientation',
+  render: () => html`
```
Show:

![image](https://github.com/user-attachments/assets/ae9e4000-9c02-49b3-9c3d-b2c55c6828d3)

smfonseca replied:
Updated

coraliefeil commented on packages/components/src/components/step-group/step-group.stories.ts:
```diff
@@ -44,240 +53,184 @@ export default {
 
 /**
- * Use the orientation attribute to set the axis of a step-group.
+ * Use the `orientation` attribute to set the axis of a step-group.
```
Change order to 

Size
Orientation

smfonseca replied:
Updated

coraliefeil commented on packages/components/src/components/step-group/step-group.stories.ts:
```diff
@@ -44,240 +53,184 @@ export default {
-/**
- * sd-steps are fully accessibile via keyboard.
- */
+        <sd-step size="lg" orientation="horizontal" state="disabled">
```
For size only show horizontal orientation:

![image](https://github.com/user-attachments/assets/340b1620-e5c7-4bbf-a6cb-486db028cf03)

smfonseca replied:
Updated

coraliefeil commented on packages/components/src/components/step-group/step-group.stories.ts:
```diff
@@ -44,240 +53,184 @@ export default {
 /**
- * This sample shows how to use the not-interactive attribute with content icons. The border-radius of an sd-step is larger when the not-interactive attribute is set to `true`.
- * The size of the icon is recommended to be 48px for the 'lg' size and 32px for the 'sm' size. The size can be set using font-size.
+ * Use the `active-step` attribute to set the current step in the step-group.
```
* Use the `active-step` attribute to set the current step.

coraliefeil replied:
Show:

![image](https://github.com/user-attachments/assets/8a02d1bf-094e-4d8d-8d56-5ae8f120a289)

coraliefeil commented on packages/components/src/components/step-group/step-group.stories.ts:
```diff
@@ -44,240 +53,184 @@ export default {
-  }
+    </div>
+  `
 };
```
How could we show that instead of numbers non-interactive could also contain a content icon?

![image](https://github.com/user-attachments/assets/211b4a19-24b1-4ad6-8a05-158e2806334b)

smfonseca replied:
I've created the icon story

coraliefeil commented on packages/components/src/components/step-group/step-group.stories.ts:
```diff
@@ -44,240 +53,184 @@ export default {
+ */
+
+export const notInteractive = {
+  name: 'Not Interactive',
```
We already have an issue to fix the center alignment of circle and label below. Right, @Vahid1919 ?

coraliefeil commented:
> Could we make it more "speaking" in the examples?
> 
> This would be one for `non-interactive` (including icons):
> 
> ```
> 1. Termin vereinbaren
> Lassen Sie sich von den Experten in unseren Partnerbanken beraten und finden Sie den richtigen Fondssparplan.
> 2. Fonds auswählen & Fondssparplan abschließen
> Wählen Sie aus einer Vielzahl von Fonds den passenden für Ihren Fondssparplan aus.
> 3. Depot eröffnen
> Eröffnen Sie gemeinsam mit Ihrem Bankberater Ihr eigenes Wertpapierdepot.
> ```
> 
> ![CleanShot 2024-08-09 at 11 06 22](https://private-user-images.githubusercontent.com/26542182/356534857-6a5c58eb-96bf-4e52-bb40-a4e54f5a3c1e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUzOTUzNTQsIm5iZiI6MTcyNTM5NTA1NCwicGF0aCI6Ii8yNjU0MjE4Mi8zNTY1MzQ4NTctNmE1YzU4ZWItOTZiZi00ZTUyLWJiNDAtYTRlNTRmNWEzYzFlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTAzVDIwMjQxNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYyNzZkZjVlZWNjODhjMGZkYTA4NTVhMDRmOWYwMzRhYzU3NDM0NDJmMGVjYWEyZGYyZTNjMjMzZmM1YjA0YjImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.EFCAVKR8qoiLBagWnfzDO8HEJ-1JK3o-b82Iz9uihFI)
> 
> And maybe for non-interactive we only use the "headlines":
> 
> ```
> 1. Termin vereinbaren
> 2. Fonds auswählen & Fondssparplan abschließen
> 3. Depot eröffnen
> ```
> 
> Ideally in English – but I have no idea regarding fitting translations here. 😆

@smfonseca I’ve added the [template here](https://www.figma.com/design/YDktJcseQIIQbsuCpoKS4V/Component-Docs?node-id=3298-8134&node-type=SECTION&t=a2qdDwUmahWYPk6r-11)

yoezlem commented on packages/components/src/components/step-group/step-group.stories.ts:
```diff
@@ -1,15 +1,24 @@
+/**
+ * Used as navigation bar that guides users through the steps of a process or task
+ *
+ * **Related templates:**
+ * - [sd-step-group](?path=/docs/templates-step-group--docs)
+ *
+ * **Related components:**
+ * - [sd-step](?path=/docs/components-step--docs)
```
```suggestion
 * **Related Components:**
  * - [sd-step](?path=/docs/components-step--docs)
 *
 * **Related Templates:**
 * - [sd-step-group](?path=/docs/templates-step-group--docs)
```

yoezlem commented on packages/components/src/components/step-group/step-group.test.stories.ts:
```diff
@@ -0,0 +1,304 @@
+      name: 'default',
+      value: `
+        <sd-step size="lg" orientation="horizontal" state="default">
+          <p slot="label">Lorem ipsum dolor sit</p>
```
```suggestion
          <p slot="label">Step 1</p>
```

yoezlem commented on packages/components/src/components/step-group/step-group.test.stories.ts:
```diff
@@ -0,0 +1,304 @@
+        </sd-step>
+
+        <sd-step size="lg" orientation="horizontal" state="current">
+          <p slot="label">Exercitation ullamco laboris</p>
```
```suggestion
          <p slot="label">Step 2</p>
```

yoezlem commented on packages/components/src/components/step-group/step-group.test.stories.ts:
```diff
@@ -0,0 +1,304 @@
+          name: 'default',
+          value: `
+            <sd-step size="lg" orientation="horizontal" state="default">
+              <p slot="label">Lorem ipsum dolor sit</p>
```
```suggestion
              <p slot="label">Step 1</p>
```

yoezlem commented on packages/components/src/components/step-group/step-group.test.stories.ts:
```diff
@@ -0,0 +1,304 @@
+            </sd-step>
+
+            <sd-step size="lg" orientation="horizontal" state="current">
+              <p slot="label">Exercitation ullamco laboris</p>
```
```suggestion
              <p slot="label">Step 2</p>
```

yoezlem commented on packages/components/src/templates/step-group.stories.ts:
```diff
@@ -0,0 +1,52 @@
+  render: () => html`
+    <sd-step-group id="set-active" size="lg" orientation="horizontal" active-step="0">
+      <sd-step size="lg" orientation="horizontal">
+        <p slot="label">Lorem ipsum dolor sit</p>
```
```suggestion
        <p slot="label">Step 1</p>
```

yoezlem commented on packages/components/src/templates/step-group.stories.ts:
```diff
@@ -0,0 +1,52 @@
+      </sd-step>
+
+      <sd-step size="lg" orientation="horizontal">
+        <p slot="label">Reprehenderit qui in e name</p>
```
```suggestion
        <p slot="label">Step 3</p>
```

yoezlem reviewed (changes requested):
Change the order to:
- Size
- Orientation

yoezlem reviewed (changes requested):
Icon Story is missing

smfonseca commented:
> > Could we make it more "speaking" in the examples?
> > This would be one for `non-interactive` (including icons):
> > ```
> > 1. Termin vereinbaren
> > Lassen Sie sich von den Experten in unseren Partnerbanken beraten und finden Sie den richtigen Fondssparplan.
> > 2. Fonds auswählen & Fondssparplan abschließen
> > Wählen Sie aus einer Vielzahl von Fonds den passenden für Ihren Fondssparplan aus.
> > 3. Depot eröffnen
> > Eröffnen Sie gemeinsam mit Ihrem Bankberater Ihr eigenes Wertpapierdepot.
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > ![CleanShot 2024-08-09 at 11 06 22](https://private-user-images.githubusercontent.com/26542182/356534857-6a5c58eb-96bf-4e52-bb40-a4e54f5a3c1e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUzOTUzNTQsIm5iZiI6MTcyNTM5NTA1NCwicGF0aCI6Ii8yNjU0MjE4Mi8zNTY1MzQ4NTctNmE1YzU4ZWItOTZiZi00ZTUyLWJiNDAtYTRlNTRmNWEzYzFlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTAzVDIwMjQxNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYyNzZkZjVlZWNjODhjMGZkYTA4NTVhMDRmOWYwMzRhYzU3NDM0NDJmMGVjYWEyZGYyZTNjMjMzZmM1YjA0YjImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.EFCAVKR8qoiLBagWnfzDO8HEJ-1JK3o-b82Iz9uihFI)
> > And maybe for non-interactive we only use the "headlines":
> > ```
> > 1. Termin vereinbaren
> > 2. Fonds auswählen & Fondssparplan abschließen
> > 3. Depot eröffnen
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Ideally in English – but I have no idea regarding fitting translations here. 😆
> 
> @smfonseca I’ve added the [template here](https://www.figma.com/design/YDktJcseQIIQbsuCpoKS4V/Component-Docs?node-id=3298-8134&node-type=SECTION&t=a2qdDwUmahWYPk6r-11)

@coraliefeil template is updated accordingly

smfonseca commented:
> Icon Story is missing

Added

smfonseca commented:
> Change the order to:
> 
> * Size
> * Orientation

Updated

Vahid1919 commented on packages/components/src/components/step-group/step-group.stories.ts:
```diff
@@ -44,240 +53,160 @@ export default {
+export const Size = {
+  name: 'Size',
+  render: () => html`
+    <div class="space-y-5">
```
This is more for general readability, but can we increase the gap between the two step groups here for the size story: 
<img width="966" alt="image" src="https://github.com/user-attachments/assets/ee77b5e6-09b8-4b06-b249-badef1447582">

Vahid1919 commented on packages/components/src/components/step-group/step-group.stories.ts:
```diff
@@ -44,240 +53,160 @@ export default {
-      </script>
-    `;
-  }
+      <div class="flex gap-12 h-[20vh] pl-[56px]">
```
```suggestion
      <div class="flex gap-12 h-[20em] pl-[56px]">
```

The height is not showing propery:
<img width="942" alt="image" src="https://github.com/user-attachments/assets/f3176848-3dda-4438-beb8-4cbbeda14f0a">

Something like 20em is much closer to the figma docs: 
<img width="1440" alt="image" src="https://github.com/user-attachments/assets/75ed98d9-c8b3-4ca3-bb40-c11950e04abd">

smfonseca replied:
Updated

coraliefeil commented:
> > > Could we make it more "speaking" in the examples?
> > > This would be one for `non-interactive` (including icons):
> > > ```
> > > 1. Termin vereinbaren
> > > Lassen Sie sich von den Experten in unseren Partnerbanken beraten und finden Sie den richtigen Fondssparplan.
> > > 2. Fonds auswählen & Fondssparplan abschließen
> > > Wählen Sie aus einer Vielzahl von Fonds den passenden für Ihren Fondssparplan aus.
> > > 3. Depot eröffnen
> > > Eröffnen Sie gemeinsam mit Ihrem Bankberater Ihr eigenes Wertpapierdepot.
> > > ```
> > > 
> > > 
> > >     
> > >       
> > >     
> > > 
> > >       
> > >     
> > > 
> > >     
> > >   
> > > ![CleanShot 2024-08-09 at 11 06 22](https://private-user-images.githubusercontent.com/26542182/356534857-6a5c58eb-96bf-4e52-bb40-a4e54f5a3c1e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUzOTUzNTQsIm5iZiI6MTcyNTM5NTA1NCwicGF0aCI6Ii8yNjU0MjE4Mi8zNTY1MzQ4NTctNmE1YzU4ZWItOTZiZi00ZTUyLWJiNDAtYTRlNTRmNWEzYzFlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTAzVDIwMjQxNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYyNzZkZjVlZWNjODhjMGZkYTA4NTVhMDRmOWYwMzRhYzU3NDM0NDJmMGVjYWEyZGYyZTNjMjMzZmM1YjA0YjImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.EFCAVKR8qoiLBagWnfzDO8HEJ-1JK3o-b82Iz9uihFI)
> > > And maybe for non-interactive we only use the "headlines":
> > > ```
> > > 1. Termin vereinbaren
> > > 2. Fonds auswählen & Fondssparplan abschließen
> > > 3. Depot eröffnen
> > > ```
> > > 
> > > 
> > >     
> > >       
> > >     
> > > 
> > >       
> > >     
> > > 
> > >     
> > >   
> > > Ideally in English – but I have no idea regarding fitting translations here. 😆
> > 
> > 
> > @smfonseca I’ve added the [template here](https://www.figma.com/design/YDktJcseQIIQbsuCpoKS4V/Component-Docs?node-id=3298-8134&node-type=SECTION&t=a2qdDwUmahWYPk6r-11)
> 
> @coraliefeil template is updated accordingly

Pls add Numbers to title and show descriptions underneath
![image](https://github.com/user-attachments/assets/0be0d668-0b5e-43d5-9f12-40bef3868998)

smfonseca commented:
> > > > Could we make it more "speaking" in the examples?
> > > > This would be one for `non-interactive` (including icons):
> > > > ```
> > > > 1. Termin vereinbaren
> > > > Lassen Sie sich von den Experten in unseren Partnerbanken beraten und finden Sie den richtigen Fondssparplan.
> > > > 2. Fonds auswählen & Fondssparplan abschließen
> > > > Wählen Sie aus einer Vielzahl von Fonds den passenden für Ihren Fondssparplan aus.
> > > > 3. Depot eröffnen
> > > > Eröffnen Sie gemeinsam mit Ihrem Bankberater Ihr eigenes Wertpapierdepot.
> > > > ```
> > > > 
> > > > 
> > > >     
> > > >       
> > > >     
> > > > 
> > > >       
> > > >     
> > > > 
> > > >     
> > > >   
> > > > ![CleanShot 2024-08-09 at 11 06 22](https://private-user-images.githubusercontent.com/26542182/356534857-6a5c58eb-96bf-4e52-bb40-a4e54f5a3c1e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUzOTUzNTQsIm5iZiI6MTcyNTM5NTA1NCwicGF0aCI6Ii8yNjU0MjE4Mi8zNTY1MzQ4NTctNmE1YzU4ZWItOTZiZi00ZTUyLWJiNDAtYTRlNTRmNWEzYzFlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTAzVDIwMjQxNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYyNzZkZjVlZWNjODhjMGZkYTA4NTVhMDRmOWYwMzRhYzU3NDM0NDJmMGVjYWEyZGYyZTNjMjMzZmM1YjA0YjImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.EFCAVKR8qoiLBagWnfzDO8HEJ-1JK3o-b82Iz9uihFI)
> > > > And maybe for non-interactive we only use the "headlines":
> > > > ```
> > > > 1. Termin vereinbaren
> > > > 2. Fonds auswählen & Fondssparplan abschließen
> > > > 3. Depot eröffnen
> > > > ```
> > > > 
> > > > 
> > > >     
> > > >       
> > > >     
> > > > 
> > > >       
> > > >     
> > > > 
> > > >     
> > > >   
> > > > Ideally in English – but I have no idea regarding fitting translations here. 😆
> > > 
> > > 
> > > @smfonseca I’ve added the [template here](https://www.figma.com/design/YDktJcseQIIQbsuCpoKS4V/Component-Docs?node-id=3298-8134&node-type=SECTION&t=a2qdDwUmahWYPk6r-11)
> > 
> > 
> > @coraliefeil template is updated accordingly
> 
> Pls add Numbers to title and show descriptions underneath ![image](https://private-user-images.githubusercontent.com/41894356/364680220-0be0d668-0b5e-43d5-9f12-40bef3868998.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU1MjUxMjMsIm5iZiI6MTcyNTUyNDgyMywicGF0aCI6Ii80MTg5NDM1Ni8zNjQ2ODAyMjAtMGJlMGQ2NjgtMGI1ZS00M2Q1LTlmMTItNDBiZWYzODY4OTk4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTA1VDA4MjcwM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU0MDQzNmM2YmQ1MTlmYjI0M2U4NDNjZDMyNGU2N2IxNmQ1YjI3NjU2MWM0MWI1MmI1OTg1MDhkODJmMzViNjUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.nEM0UwbKMZ3xGoQ-UM8Q5QW3VnOZElRnjCcK17Ljoak)

Updated

coraliefeil commented:
> > > > > Could we make it more "speaking" in the examples?
> > > > > This would be one for `non-interactive` (including icons):
> > > > > ```
> > > > > 1. Termin vereinbaren
> > > > > Lassen Sie sich von den Experten in unseren Partnerbanken beraten und finden Sie den richtigen Fondssparplan.
> > > > > 2. Fonds auswählen & Fondssparplan abschließen
> > > > > Wählen Sie aus einer Vielzahl von Fonds den passenden für Ihren Fondssparplan aus.
> > > > > 3. Depot eröffnen
> > > > > Eröffnen Sie gemeinsam mit Ihrem Bankberater Ihr eigenes Wertpapierdepot.
> > > > > ```
> > > > > 
> > > > > 
> > > > >     
> > > > >       
> > > > >     
> > > > > 
> > > > >       
> > > > >     
> > > > > 
> > > > >     
> > > > >   
> > > > > ![CleanShot 2024-08-09 at 11 06 22](https://private-user-images.githubusercontent.com/26542182/356534857-6a5c58eb-96bf-4e52-bb40-a4e54f5a3c1e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUzOTUzNTQsIm5iZiI6MTcyNTM5NTA1NCwicGF0aCI6Ii8yNjU0MjE4Mi8zNTY1MzQ4NTctNmE1YzU4ZWItOTZiZi00ZTUyLWJiNDAtYTRlNTRmNWEzYzFlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTAzVDIwMjQxNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYyNzZkZjVlZWNjODhjMGZkYTA4NTVhMDRmOWYwMzRhYzU3NDM0NDJmMGVjYWEyZGYyZTNjMjMzZmM1YjA0YjImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.EFCAVKR8qoiLBagWnfzDO8HEJ-1JK3o-b82Iz9uihFI)
> > > > > And maybe for non-interactive we only use the "headlines":
> > > > > ```
> > > > > 1. Termin vereinbaren
> > > > > 2. Fonds auswählen & Fondssparplan abschließen
> > > > > 3. Depot eröffnen
> > > > > ```
> > > > > 
> > > > > 
> > > > >     
> > > > >       
> > > > >     
> > > > > 
> > > > >       
> > > > >     
> > > > > 
> > > > >     
> > > > >   
> > > > > Ideally in English – but I have no idea regarding fitting translations here. 😆
> > > > 
> > > > 
> > > > @smfonseca I’ve added the [template here](https://www.figma.com/design/YDktJcseQIIQbsuCpoKS4V/Component-Docs?node-id=3298-8134&node-type=SECTION&t=a2qdDwUmahWYPk6r-11)
> > > 
> > > 
> > > @coraliefeil template is updated accordingly
> > 
> > 
> > Pls add Numbers to title and show descriptions underneath ![image](https://private-user-images.githubusercontent.com/41894356/364680220-0be0d668-0b5e-43d5-9f12-40bef3868998.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU1MjUxMjMsIm5iZiI6MTcyNTUyNDgyMywicGF0aCI6Ii80MTg5NDM1Ni8zNjQ2ODAyMjAtMGJlMGQ2NjgtMGI1ZS00M2Q1LTlmMTItNDBiZWYzODY4OTk4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTA1VDA4MjcwM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU0MDQzNmM2YmQ1MTlmYjI0M2U4NDNjZDMyNGU2N2IxNmQ1YjI3NjU2MWM0MWI1MmI1OTg1MDhkODJmMzViNjUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.nEM0UwbKMZ3xGoQ-UM8Q5QW3VnOZElRnjCcK17Ljoak)
> 
> Updated

Add description, pls

![image](https://github.com/user-attachments/assets/005e6bc1-a19a-497a-9216-1887d759525a)

smfonseca commented:
> > > > > > Could we make it more "speaking" in the examples?
> > > > > > This would be one for `non-interactive` (including icons):
> > > > > > ```
> > > > > > 1. Termin vereinbaren
> > > > > > Lassen Sie sich von den Experten in unseren Partnerbanken beraten und finden Sie den richtigen Fondssparplan.
> > > > > > 2. Fonds auswählen & Fondssparplan abschließen
> > > > > > Wählen Sie aus einer Vielzahl von Fonds den passenden für Ihren Fondssparplan aus.
> > > > > > 3. Depot eröffnen
> > > > > > Eröffnen Sie gemeinsam mit Ihrem Bankberater Ihr eigenes Wertpapierdepot.
> > > > > > ```
> > > > > > 
> > > > > > 
> > > > > >     
> > > > > >       
> > > > > >     
> > > > > > 
> > > > > >       
> > > > > >     
> > > > > > 
> > > > > >     
> > > > > >   
> > > > > > ![CleanShot 2024-08-09 at 11 06 22](https://private-user-images.githubusercontent.com/26542182/356534857-6a5c58eb-96bf-4e52-bb40-a4e54f5a3c1e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUzOTUzNTQsIm5iZiI6MTcyNTM5NTA1NCwicGF0aCI6Ii8yNjU0MjE4Mi8zNTY1MzQ4NTctNmE1YzU4ZWItOTZiZi00ZTUyLWJiNDAtYTRlNTRmNWEzYzFlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTAzVDIwMjQxNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYyNzZkZjVlZWNjODhjMGZkYTA4NTVhMDRmOWYwMzRhYzU3NDM0NDJmMGVjYWEyZGYyZTNjMjMzZmM1YjA0YjImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.EFCAVKR8qoiLBagWnfzDO8HEJ-1JK3o-b82Iz9uihFI)
> > > > > > And maybe for non-interactive we only use the "headlines":
> > > > > > ```
> > > > > > 1. Termin vereinbaren
> > > > > > 2. Fonds auswählen & Fondssparplan abschließen
> > > > > > 3. Depot eröffnen
> > > > > > ```
> > > > > > 
> > > > > > 
> > > > > >     
> > > > > >       
> > > > > >     
> > > > > > 
> > > > > >       
> > > > > >     
> > > > > > 
> > > > > >     
> > > > > >   
> > > > > > Ideally in English – but I have no idea regarding fitting translations here. 😆
> > > > > 
> > > > > 
> > > > > @smfonseca I’ve added the [template here](https://www.figma.com/design/YDktJcseQIIQbsuCpoKS4V/Component-Docs?node-id=3298-8134&node-type=SECTION&t=a2qdDwUmahWYPk6r-11)
> > > > 
> > > > 
> > > > @coraliefeil template is updated accordingly
> > > 
> > > 
> > > Pls add Numbers to title and show descriptions underneath ![image](https://private-user-images.githubusercontent.com/41894356/364680220-0be0d668-0b5e-43d5-9f12-40bef3868998.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU1MjUxMjMsIm5iZiI6MTcyNTUyNDgyMywicGF0aCI6Ii80MTg5NDM1Ni8zNjQ2ODAyMjAtMGJlMGQ2NjgtMGI1ZS00M2Q1LTlmMTItNDBiZWYzODY4OTk4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTA1VDA4MjcwM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU0MDQzNmM2YmQ1MTlmYjI0M2U4NDNjZDMyNGU2N2IxNmQ1YjI3NjU2MWM0MWI1MmI1OTg1MDhkODJmMzViNjUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.nEM0UwbKMZ3xGoQ-UM8Q5QW3VnOZElRnjCcK17Ljoak)
> > 
> > 
> > Updated
> 
> Add description, pls
> 
> ![image](https://private-user-images.githubusercontent.com/41894356/364722321-005e6bc1-a19a-497a-9216-1887d759525a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU1MzE0NDMsIm5iZiI6MTcyNTUzMTE0MywicGF0aCI6Ii80MTg5NDM1Ni8zNjQ3MjIzMjEtMDA1ZTZiYzEtYTE5YS00OTdhLTkyMTYtMTg4N2Q3NTk1MjVhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTA1VDEwMTIyM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY1MDA5NTAyMmQ4ZTQ2NjhjMjNlNWE4MDY5NDA3NGFhMjk4OWJjOTFkZDQ0YmY3NTEwZDBmYzE5OWQ1NDUyNWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.-vp9W7KEmB38HRTEOV7H1f7aVLPbipeTjFcJ6dY_bQM)

Done. Sorry, misread the initial request

coraliefeil commented:
Approved <3
